__tests__/app.test.js

```diff
@@ -165,5 +165,96 @@ describe("CORE:",() => {
                 })
             })
         })
+        describe("POST:", () => {
+            it("201: attaches new comment to an article", () => {
+                return request(app)
+                .post("/api/articles/1/comments")
+                .send({
+                    body:"meow",
+                    username: "icellusedkars",
+                })
+                .expect(201)
+                .then((response) => {
+                    const {
+                        body: { comment },
+                    } = response;
+                    const newComment = comment[0]
```

Comment:
This should respond with just the object, not an array. In your model if you return rows[0] you will fix this 👍